### PR TITLE
Jdp230601 15

### PR DESCRIPTION
**Pull request by ZuzannJanik**
### Jira ticket: Jdp230601 15
https://projects.kodilla.com/secure/RapidBoard.jspa?rapidView=217&projectKey=JDP230601&view=detail&selectedIssue=JDP230601-15

### What has changed:
- I created a User entity with variables
- I defined a dependency with entity Cart

### What reviewers should know:
I created the variables based on the established scheme. However, it seems that the token validation variable may be different depending on the token validation method used.

### CODE REVIEW

**Review by piotr-grosicki — CHANGES_REQUESTED** [review]
projekt sie nie buduje

**Comment by piotr-grosicki** [comment]
konflikty, https://www.jetbrains.com/help/idea/resolving-conflicts.html

**Comment by ZuzannJanik** [comment]
Konflikty naprawione

File: build.gradle
Comment (konradpieczynski) [inline build.gradle:1]: Nie potrzebne.

**Review by konradpieczynski — CHANGES_REQUESTED** [review]
Nie potrzebnie dodana linijka w build.gradle

File: build.gradle
Comment (ZuzannJanik) [reply build.gradle:1]: Usunięte.

**Review by konradpieczynski — APPROVED** [review]

**Pull request merged**